libkernelflinger/vars.c

```diff
@@ -791,6 +791,21 @@ char *get_device_id(void)
 }
 #endif
 
+/* This is a WA, we use the asset field of type_chassis to transfer the
+ * host ip to bootloader.*/
+char *get_host_ip(void)
+{
+	static char host_ip[ANDROID_PROP_VALUE_MAX];
```

Comment:
why use static? what about define local variable char host_ip[ANDROID_PROP_VALUE_MAX] in fastboot_get_host_ip(), then call get_host_ip(char *host_ip) to get host ip.

Reply:
This can be consistent with other functions implemented in vars.c, such as get_device_id, get_property_bootloader etc.